hnn_core/cell_response.py

```diff
@@ -35,6 +35,11 @@ class CellResponse(object):
     cell_type_names : list
         List of unique cell type names that are explicitly modeled in the
         network
+    vsec : list (n_trials,) of dict, shape
```

Comment:
```suggestion
    vsec : list of dict, shape (n_trials,)
```
There are a few entries above that need a similar wording fix.

Reply:
We could just remove "shape"? and leave the length next to the appropriate descriptor